### PR TITLE
New app: birdbyt

### PR DESCRIPTION
**Pull request by bsweger**
# Description
Birdbyte uses Cornell's eBird API to display a random bird sighting near a specified location. 

The app gets 100 recent bird sightings and caches them for an hour. Each render of the app chooses one of the 100 sightings at random and formats it for the Tidbyt screen. 

Birdbyt also tries to determine whether or not the chosen sighting is a designated "notable sighting." In the case of a "notable" sighting, the default calm purple bird image is replaced by a more excitable purple bird.

Config options:
- Location
- Search radius
- How many days back to look for bird sightings
- Whether or not to include unverified bird sightings

# Copilot

### Summary
🐦📜📝

This pull request adds the Birdbyt app, which displays bird sightings data from the eBird API on the Tidbyt device. It includes the main script file `birdbyt.star`, the manifest file `manifest.yaml`, and the README file `README.md`, which provide the app logic, metadata, and documentation respectively.

> _`Birdbyt` app is born_
> _Fetching and showing bird data_
> _A spring delight_

### Walkthrough
* Create and register the Birdbyt app to display bird sightings data from the eBird API ([link](https://github.com/tidbyt/community/pull/1555/files?diff=unified&w=0#diff-32439ab0595350d4e094ab3f2167cb4db44da23a22ebc046f270068a1c21571aR1-R396), [link](https://github.com/tidbyt/community/pull/1555/files?diff=unified&w=0#diff-e9502e76d3fe08348983e518212ceb8f30d900abf91b52680814bf6406820880R1-R8))
* Document the app and its features for potential users and contributors ([link](https://github.com/tidbyt/community/pull/1555/files?diff=unified&w=0#diff-cf495c2db97e801b5969c20aaa9025952d7e710acad4fe907ce5a09164cccd3aR1-R15))

### CODE REVIEW

**Comment by bsweger** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/birdbyt/birdbyt.star [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+def main(config):
+    """Update config.
+
+    Args:
+      config: config dict passed from the app
+
+    Returns:
+      rendered WebP image for Tidbyt display
+    """
+    ebird_key = secret.decrypt(EBIRD_API_KEY) or config.get("ebird_api_key")
```
Comment (matslina) [inline apps/birdbyt/birdbyt.star:184]:
```suggestion
    random.seed(time.now().unix // 10)
    ebird_key = secret.decrypt(EBIRD_API_KEY) or config.get("ebird_api_key")
```

Comment (matslina) [reply apps/birdbyt/birdbyt.star:184]: This makes the app pick the same bird for all installations of the app over a 10 second window. Makes it easier for our backend to cache generated graphics properly. Hope you don't mind!

**Review by matslina — DISMISSED** [review]

File: apps/birdbyt/birdbyt.star [hunk 2]
```diff
@@ -0,0 +1,396 @@
+"""
+Applet: Birdbyt
+Summary: Show nearby bird sightings
+Description: Displays a random bird sighting near a specific location.
+Author: Becky Sweger
+"""
+
+load("cache.star", "cache")
```
Comment (matslina) [inline apps/birdbyt/birdbyt.star:8]: Wanted to mention that the http module now has built in caching. Just pass `ttl_seconds` to `http.get()`. Might be worth migrating over to that in a later PR. It'll simplify the code a bit, and also future proofs things a bit as the cache module may change a bit down the road.

Comment (bsweger) [reply apps/birdbyt/birdbyt.star:8]: Sounds great--will to a follow-up PR after the app is published!

File: apps/birdbyt/birdbyt.star (hunk 1 above)
Comment (bsweger) [reply apps/birdbyt/birdbyt.star:185]: @matslina quick question: will this make the app pick the same bird for all installations of the app even they're using different location parameters to get birds? or would it be the same bird for all installations that share the same config? Thanks!

**Pull request merged**